app/src/main/java/fr/free/nrw/commons/SettingsActivity.java

```diff
@@ -68,7 +79,12 @@ protected void onPause() {
 
     @Override
     public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key) {
-
+        if(key.equals("theme")){
+            //Finish current activity and tart new one with selected theme
```

Comment:
Typo in word "start"

Also kinda related - maybe just a small thing but using spaces between slashes and comment text, as per [Google's Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace)